src/test/java/com/google/cloud/spanner/jdbc/it/ITJdbcSimpleStatementsTest.java

```diff
@@ -102,12 +102,6 @@ public void testSelect1PreparedStatement() throws SQLException {
 
   @Test
   public void testPreparedStatement() throws SQLException {
-    // skipping the test when dialect is POSTGRESQL because of exception below
-    // INVALID_ARGUMENT: io.grpc.StatusRuntimeException: INVALID_ARGUMENT: Statements with set
-    // operations in subqueries are not supported
-    assumeFalse(
-        "select array of structs is not supported on POSTGRESQL",
-        dialect.dialect == Dialect.POSTGRESQL);
```

Comment:
This is unrelated to this change, but cleaning it up now that we're touching this file anyways; This limitation no longer applies to PostgreSQL-dialect databases.